README.md

```diff
@@ -63,9 +63,9 @@ The next time you enter the dev shell, the updated binaries will be downloaded a
 
 ## Including other versions of Holochain
 
-By default the Holonix input references the `main` branch of the Holonix repository. Whatever the current revision of that branch at the moment of invoking the dev shell for the first time is will be written to the `flake.lock` file. The `main` branch uses version 0.4 of Holochain and compatible versions of scaffolding etc.
+By default the Holonix input references the `main` branch of the Holonix repository. Whatever the current revision of that branch at the moment of invoking the dev shell for the first time is will be written to the `flake.lock` file. The `main` branch uses version 0.5 of Holochain and compatible versions of scaffolding etc.
 
-If you want to develop using the 0.3 branch of Holochain instead, you can create the corresponding Nix flake running this command inside your project folder:
+If you want to develop using the 0.3 or 0.4 branch of Holochain instead, you can create the corresponding Nix flake running this command inside your project folder:
```

Comment:
This is not accurate against the command below. It specifically gives you the 0.4 version. I think until 0.4 is released we should keep recommending 0.3 here

Reply:
oh, but isn't `main` the default branch in `holochain/holonix`, and isn't it tracking 0.5?

Reply:
```suggestion
If you want to develop using the 0.3 branch (recommended) or 0.4 branch (RC) of Holochain instead, you can create the corresponding Nix flake running this command inside your project folder:
```

Reply:
The top part is correct, not the "0.3 or 0.4" part